example/src/main/java/org/apache/rocketmq/example/simple/PushConsumer.java

```diff
@@ -26,10 +26,14 @@
 import org.apache.rocketmq.common.message.MessageExt;
 
 public class PushConsumer {
-
+    public static final String TOPIC = "TopicTest";
+    public static final String CONSUMER_GROUP = "CID_JODIE_1";
+    public static final String NAMESRV_ADDR = "127.0.0.1:9876";
     public static void main(String[] args) throws InterruptedException, MQClientException {
-        DefaultMQPushConsumer consumer = new DefaultMQPushConsumer("CID_JODIE_1");
-        consumer.subscribe("TopicTest", "*");
+
+        DefaultMQPushConsumer consumer = new DefaultMQPushConsumer(CONSUMER_GROUP);
+        consumer.setNamesrvAddr(NAMESRV_ADDR);
```

Comment:
It's an example program and users may prefer to fill name server address through an environment variable or java options...hardcoding one may not be helpful

Reply:
For most people who have just used RocketMQ, they just want to successfully run the PushConsumer, but they will get a "connect to null failed" message. In addition, namesrvAddr is also hard coded in PullConsumer, so I think hard coding namesrvAddr is helpful for novices who have just learned RocketMQ.

Reply:
yes, i think explicitly define NamesrvAddr will help new users to avoid unexpected null errors